docs/documentation/server_development/topics/user-storage/javaee.adoc

```diff
@@ -1,87 +1,16 @@
 
 === Leveraging Jakarta EE
 
-The user storage providers can be packaged within any Jakarta EE component if you set up the `META-INF/services`
-file correctly to point to your providers. For example, if your provider needs to use third-party libraries, you
-can package up your provider within an EAR and store these third-party libraries in the `lib/` directory of the EAR.
-Also note that provider JARs can make use of the `jboss-deployment-structure.xml` file that EJBs, WARS, and EARs
-can use in a {appserver_name} environment. For more details on this file, see the {appserver_name} documentation. It
-allows you to pull in external dependencies among other fine-grained actions.
+Since its version 20, Keycloak relies only on Quarkus. And unlike WildFly, Quarkus is not an Application Server. Read more about this at https://www.keycloak.org/migration/migrating-to-quarkus#_quarkus_is_not_an_application_server .
```

Comment:
```suggestion
Since version 20, Keycloak relies only on Quarkus. Unlike WildFly, Quarkus is not an Application Server. For more detail, see https://www.keycloak.org/migration/migrating-to-quarkus#_quarkus_is_not_an_application_server .
```

Reply:
Suggestion accepted.

Thanks, @andymunro !